lib/cloudregister/registerutils.py

```diff
@@ -136,15 +136,14 @@ def clear_new_registration_flag():
 
 
 # ----------------------------------------------------------------------------
-def credentials_files_are_equal():
-    """Compare all the credentials files and make sure they have the same
+def credentials_files_are_equal(in_credential):
```

Comment:
Please rename the argument to "repo_credentials"

Reply:
done